cmd/juju/charmhub/download.go

```diff
@@ -230,7 +230,12 @@ func (c *downloadCommand) Run(cmdContext *cmd.Context) error {
 		path = fmt.Sprintf("%s%s.%s", entity.Name, short, entityType)
```

Comment:
Right now we add the charm's sha to the name to create a filename if not specified. It could be easier to read the file name if we add the charm's revision instead?

`ubuntu_21.charm` rather than `ubuntu_7272c62.charm`

Reply:
That's a good call, done.